src/schema/rules/checks/hints.yaml

```
@@ -34,6 +34,18 @@ TooFewAuthors:
   checks:
     - length(json.Authors) > 1
 
+# 115
+EmptyDatasetName:
+  issue:
+    code: EMPTY_DATASET_NAME
+    message: |
+      The Name field of dataset_description.json is present but empty of visible characters.
+    level: warning
+  selectors:
+    - path == '/dataset_description.json'
+  checks:
+    - match(json.name, '\S')
```

Needs to be `json.Name`. And we should not duplicate the error if `Name` is missing.

```suggestion
  selectors:
    - path == '/dataset_description.json'
    - type(json.Name) != 'null'
  checks:
    - match(json.Name, '\S')
```